conf/defaults.ini

```diff
@@ -230,6 +230,9 @@ application_insights_connection_string =
 # Optional. Specifies an Application Insights endpoint URL where the endpoint string is wrapped in backticks ``.
 application_insights_endpoint_url =
 
+# Controls if the UI contains any links to user feedback forms
+feedback_links_enabled = true
```

Comment:
Does it make sense to make this global? What is the usecase to disable it?

Reply:
Mostly a backup in case CX get's complaints from customers who do not want this link